File: Data Structures & Algorithms/sort-colors/submission-1.py
class Solution:
    def sortColors(self, nums: List[int]) -> None:
        """
        Do not return anything, modify nums in-place instead.
        """
        colors = [0, 0, 0]

        for n in nums:
            colors[n] += 1

        i = 0

        for n in range(len(colors)):
            for j in range(colors[n]):
                nums[i] = n
                i += 1

        return nums

File: Data Structures & Algorithms/binary-search/submission-1.java
class Solution {
    public int search(int[] nums, int target) {
        int L = 0;
        int R = nums.length - 1;

        while (L <= R) {
            int mid = (R + L) / 2;
            if (target > nums[mid]) {
                L = mid + 1;
            } else if (target < nums[mid]) {
                R = mid - 1;
            } else {
                return mid;
            }
        }

        return -1;
    }
}
